google/cloud/spanner/instance_admin_connection.h

```diff
@@ -237,16 +237,34 @@ std::shared_ptr<InstanceAdminConnection> MakeInstanceAdminConnection(
 
 namespace spanner_internal {
 inline namespace SPANNER_CLIENT_NS {
-std::shared_ptr<spanner::InstanceAdminConnection> MakeInstanceAdminConnection(
-    std::shared_ptr<InstanceAdminStub> base_stub,
-    spanner::ConnectionOptions const& options);
 
+/**
+ * Returns an InstanceAdminConnection object that can be used for interacting
+ * with Cloud Spanner's admin APIs.
+ *
+ * The returned connection object should not be used directly, rather it should
```

Comment:
s/directly, rather/directly; instead/

Reply:
Fixed in https://github.com/googleapis/google-cloud-cpp/pull/6029